index.html

```diff
@@ -1,11 +1,36 @@
 <!DOCTYPE html>
 <html lang="en">
 <head>
+    <link rel="stylesheet" type="text/css" href="style/index.css">
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>Document</title>
+    <title> Document</title>
+    <h1>Karl Stone-Seigler</h1>
 </head>
 <body>
+
+    <nav>
```

Comment:
nice job with your navigation! The only thing I'd say is try to have your anchor tags link to the corresponding pages (: